engine/engine/src/engine.cpp

```diff
@@ -1509,6 +1567,8 @@ namespace dmEngine
 
     static void Reboot(HEngine engine, dmSystemDDF::Reboot* reboot)
     {
+        dmLogWarning("ENGINE REBOOT");
```

Comment:
I guess it's ok with a log here but why a warning? Something left over from testing?

Reply:
no, it shouldn't be there (it's a debug print)